sql/sql_parse.cc

```diff
@@ -2572,7 +2572,7 @@ static bool lock_tables_for_backup(THD *thd)
   @return FALSE in case of success, TRUE in case of error.
 */
 
-static bool lock_binlog_for_backup(THD *thd)
+bool lock_binlog_for_backup(THD *thd)
```

Comment:
The rules AFAIK are common, but this is an area that not a lot of people tread into. Basically any change to the core server or core engines needs a blueprint or bug. This often means that you end up with both JIRA issues and launchpad blueprint/bug for the same issue. This comes into play with TokuDB, BakupPlugin, MyRocks, etc...